app/interfaces/api/entities/ccr/adapted_basic_fee.rb

```diff
@@ -22,8 +22,12 @@ def fee_quantity_for(fee_type_unique_code)
           fee_for(fee_type_unique_code)&.quantity.to_i
         end
 
-        def ppe
-          fee_quantity_for('BAPPE')
+        def adapted_ppe
+          if object.discontinuance?
+            object.prosecution_evidence == true ? 1 : 0
```

Comment:
this should be able to take `object.prosecution_evidence? ? 1 : 0`

Reply:
👍

Reply:
👍